guides/line-management/promotions.md

```diff
@@ -0,0 +1,56 @@
+# Promotions
+
+This guidances is intended to help line managers to facilitate the career progression of the people they line manage.
```

Comment:
There's some changes we could make here to make this read clearer, I think:
- using simpler words where possible, e.g. facilitate -> help, utilise -> use
- writing to the reader directly (using "you" for the line manager reading the doc)
- using smaller sentences

There's a few examples throughout the doc, but as an example for this line:
```suggestion
This guidance is written for line managers. It'll help you understand how promotions work at Made Tech. 

You're responsible for supporting people you line manage when they want to get promoted.
```